src/edu/csus/ecs/pc2/core/model/Filter.java

```diff
@@ -1,54 +1,59 @@
-// Copyright (C) 1989-2019 PC2 Development Team: John Clevenger, Douglas Lane, Samir Ashoo, and Troy Boudreau.
+// Copyright (C) 1989-2024 PC2 Development Team: John Clevenger, Douglas Lane, Samir Ashoo, and Troy Boudreau.
 package edu.csus.ecs.pc2.core.model;
 
 import java.io.Serializable;
+import java.util.Collection;
+import java.util.Collections;
```

Comment:
I'm confused by these?  I don't see them used?

Reply:
HashSet requires them.  Automatically added by Eclipse and can't be removed.